services/brig/src/Brig/API/Internal.hs

```diff
@@ -334,20 +337,42 @@ changeSelfEmailMaybeSend u DoNotSendEmail email = do
     ChangeEmailIdempotent -> pure ChangeEmailResponseIdempotent
     ChangeEmailNeedsActivation _ -> pure ChangeEmailResponseNeedsActivation
 
-listActivatedAccountsH :: JSON ::: Either (List UserId) (List Handle) -> Handler Response
-listActivatedAccountsH (_ ::: qry) = do
-  json <$> lift (listActivatedAccounts qry)
-
-listActivatedAccounts :: Either (List UserId) (List Handle) -> AppIO [UserAccount]
-listActivatedAccounts = \case
-  Left us -> byIds (fromList us)
-  Right hs -> do
-    us <- mapM (API.lookupHandle) (fromList hs)
-    byIds (catMaybes us)
+listActivatedAccountsH :: JSON ::: Either (List UserId) (List Handle) ::: Bool -> Handler Response
+listActivatedAccountsH (_ ::: qry ::: includePendingInvitations) = do
+  json <$> lift (listActivatedAccounts qry includePendingInvitations)
+
+listActivatedAccounts :: Either (List UserId) (List Handle) -> Bool -> AppIO [UserAccount]
+listActivatedAccounts elh includePendingInvitations = do
+  Log.debug (Log.msg $ "listActivatedAccounts: " <> show (elh, includePendingInvitations))
+  case elh of
+    Left us -> byIds (fromList us)
+    Right hs -> do
+      us <- mapM (API.lookupHandle) (fromList hs)
+      byIds (catMaybes us)
   where
-    byIds uids =
-      filter (isJust . userIdentity . accountUser)
-        <$> API.lookupAccounts uids
+    byIds :: [UserId] -> AppIO [UserAccount]
+    byIds uids = API.lookupAccounts uids >>= filterM accountValid
```

Comment:
This still doesn't  work; API.lookupAccounts only returns user accounts with activated=true. You want some accounts that are activated=false, so this logic isn't right, you need to use another (or write a new function), or adjust the existing lookupAccounts function.

Reply:
This is not valid any more: we found that it creates bigger problems to have users with `activated==false && status=PendingActivation`, and changed them to `activated==true && status==PendingActivation`.  So this line should do what it's supposed to.